src/main/java/uk/gov/hmcts/darts/transcriptions/service/impl/TranscriptionServiceImpl.java

```diff
@@ -297,6 +297,7 @@ private TranscriptionEntity saveTranscription(UserAccountEntity userAccount,
         transcription.setEndTime(transcriptionRequestDetails.getEndDateTime());
         transcription.setCreatedBy(userAccount);
         transcription.setLastModifiedBy(userAccount);
+        transcription.setIsManual(false);
```

Comment:
This Boolean will need to be parametised - please can you change, the `saveTranscriptionRequest`
https://github.com/search?q=repo%3Ahmcts%2Fdarts-api%20saveTranscriptionRequest&type=code

to introduce the new boolean separate from `TranscriptionRequestDetails` as then the automatic handler [TranscriptionRequestHandler.java](https://github.com/hmcts/darts-api/blob/2ee0701f9d71c5ef5b6c0545253d5df52f98e0da/src/main/java/uk/gov/hmcts/darts/event/service/impl/TranscriptionRequestHandler.java#L39) can pass in FALSE whereas the [TranscriptionController.java](https://github.com/hmcts/darts-api/blob/2ee0701f9d71c5ef5b6c0545253d5df52f98e0da/src/main/java/uk/gov/hmcts/darts/transcriptions/controller/TranscriptionController.java#L72) would pass in TRUE.